### ISSUE TITLE
sbt 1.1.2 suppresses info output if --info is given

### ISSUE THREAD

**Issue by sschuberth** [sbt/sbt#4285]
- [X] I've read the [CONTRIBUTING](https://github.com/sbt/sbt/blob/1.x/CONTRIBUTING.md) guidelines

## steps

```
$ cat project/build.properties
sbt.version = 1.1.2
$ sbt --info name
```

## problem

The output does not show *any* `[info]` lines.

## expectation

The output should show `[info]` lines as `--info` is given.

## notes

sbt version: 1.1.2

This works fine for me with sbt 1.1.1, so I'm wondering whether https://github.com/sbt/sbt/pull/4036 is the culprit.

**Comment by eed3si9n** [comment]
Thanks for the report.
Would you like to try sending a PR for this?

**Comment by sschuberth** [comment]
No, I'm sorry. I'm just an SBT user, not a Scala guy (and I don't intend to become one 😉 ). I'd have no idea how to fix that.
